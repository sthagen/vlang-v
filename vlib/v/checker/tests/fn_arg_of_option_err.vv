fn option_arg(x ?int) {
	println('int type: ${x}')
}

fn main() {
	option_arg(1)
}
