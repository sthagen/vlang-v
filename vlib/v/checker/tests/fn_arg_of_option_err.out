vlib/v/checker/tests/fn_arg_of_option_err.vv:1:17: error: option or result type argument is not supported currently
    1 | fn option_arg(x ?int) {
      |                 ~~~~
    2 |     println('int type: ${x}')
    3 | }